app/assets/javascripts/app/core/models/discussion.js

```diff
@@ -36,6 +36,10 @@
           var discussion = this;
 
           return DS.refresh('discussion', discussion.id);
+        },
```

Comment:
Can't you clean `addComment` now? I believe you don't need `linkInverse` since it's in *Comment*'s `afterInject`.
Does the endpoint need to be specified?
Can't you use the inject `Comment` service instead of `DS.create('comment', ...)` ?